spacy/ml/models/tok2vec.py

```diff
@@ -186,6 +189,197 @@ def make_hash_embed(index):
     return model
 
 
+def _verify_rich_config_group(
+    label: str,
+    lengths: Optional[List[int]],
+    rows: Optional[List[int]],
+    search_chars: Optional[str],
+    is_search_char_group: bool,
+) -> None:
+    if lengths is not None or rows is not None:
+        if is_search_char_group and (search_chars is None or len(search_chars) == 0):
+            raise ValueError(Errors.E1047.format(label=label))
+        if search_chars is not None and len(search_chars) > 63:
+            raise ValueError(Errors.E1048.format(label=label))
+        if lengths is None or rows is None:
+            raise ValueError(Errors.E1047.format(label=label))
+        if len(lengths) != len(rows):
+            raise ValueError(Errors.E1047.format(label=label))
+        if any([length < 1 for length in lengths]):
+            raise ValueError(Errors.E1047.format(label=label))
+    elif search_chars is not None:
+        raise ValueError(Errors.E1047.format(label=label))
+    if lengths is not None:
+        if lengths[-1] > 63:
+            raise ValueError(Errors.E1048.format(label=label))
+        if len(lengths) != len(set(lengths)) or lengths != sorted(lengths):
+            raise ValueError(Errors.E1049.format(label=label))
+
+
```

Comment:
This function contains all fields from `MultiHashEmbed.v2`. The only change required to make it backwards compatible with respect to `MultiHashEmbed.v2` would be to make `case_sensitive` optional. If so desired, then, we could replace `MultiHashEmbed.v2` and name this function `MultiHashEmbed.v3` instead of `RichMultiHashEmbed.v1`.

Reply:
Good point!